pkg/ipmi/ipmi.go

```diff
@@ -173,3 +207,44 @@ func (i *IPMI) ShutoffWatchdog() error {
 
 	return nil
 }
+
+func (e *Event) Marshall() []byte {
```

Comment:
s/Marshall/marshal/

Reply:
How about making it a private method?

Reply:
I modified the marshal method to a private method.